i18nfield/forms.py

```diff
@@ -86,7 +86,10 @@ def render(self, name: str, value, attrs=None, renderer=None) -> str:
         return mark_safe(self.format_output(output))
 
     def format_output(self, rendered_widgets) -> str:
-        return '<div class="i18n-form-group">%s</div>' % ''.join(rendered_widgets)
+        return '<div class="i18n-form-group%s">%s</div>' % (
+            ' hide-flags' if len(rendered_widgets) <= 1 else '',
```

Comment:
Since this is a library used in multiple projects I'd suggest to prefix the class name to avoid clashes

```suggestion
            ' i18n-form-hide-flags' if len(rendered_widgets) <= 1 else '',
```

Reply:
Or maybe, to be even more accurate, ``i18n-form-single-language``?

Reply:
Good point, done!